### PR TITLE
#420 Error code in README fix

### PR DESCRIPTION
**Pull request by chandresh-pancholi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Some API methods have optional parameters that can be passed. For example:
 client := github.NewClient(nil)
 
 // list recently updated repositories for org "github"
-opt := &github.RepositoryListByOrgOptions{Sort: "updated"}
+opt := &github.RepositoryListByOrgOptions{Type: "member"}
```
Comment (willnorris) [inline README.md]: I honestly don't remember what "member" means for an organization repo.  Maybe change this to "public", and then update the comment above to reflect that.  Same in doc.go

**Comment by willnorris** [comment]
Not sure why the CLA checker didn't comment when you first opened the pull request, but it does look like we need a CLA from you in order to merge this. (see instructions in googlebot comment above)

File: README.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Some API methods have optional parameters that can be passed. For example:
 client := github.NewClient(nil)
 
 // list recently updated repositories for org "github"
```
Comment (willnorris) [inline README.md]: also update the comment here so that it's accurate.  Same with the one in doc.go

**Comment by chandresh-pancholi** [comment]
I signed it!

**Comment by willnorris** [comment]
awesome, thanks!

**Pull request closed without merging**